extensions/data-transfer/portability-data-transfer-rememberthemilk/src/main/java/org/dataportabilityproject/transfer/rememberthemilk/tasks/RememberTheMilkService.java

```diff
@@ -83,8 +83,29 @@ public TaskSeries createTask(String name, String timeline, String listId) throws
             name,
             "list_id",
             listId);
-    TaskAddResponse taskAddResponse = makeRequest(params, TaskAddResponse.class);
-    return taskAddResponse.list.taskseries.get(0);
+    TaskUpdateResponse taskUpdateResponse = makeRequest(params, TaskUpdateResponse.class);
+    return taskUpdateResponse.list.taskseries.get(0);
+  }
+
+  public void completeTask(String timeline, String listId, int seriesId, int taskId)
+      throws IOException {
+    // NB: This does not transfer the complete time!  There's no method for that.  It just sets a
+    // task as being complete, and I believe that the completion time is the time that the request
```

Comment:
In other words, RTM does not support completion time in their api?

Reply:
As far as I can tell, that is the case.